internal/pkg/diff/diff_test.go

```diff
@@ -803,6 +803,6 @@ func TestResults_Format(t *testing.T) {
 
 func newProjectState(t *testing.T) *state.State {
 	t.Helper()
-	d := dependencies.NewMockedDeps()
+	d := dependencies.NewMockedDeps(t)
```

Comment:
Buffer API potrebuje `Store`, to potrebuje `EtcdClient`, ten vytvorime cez `etcdhelper.ClientForTest(t)`,
takze `MockedDeps` potrebuju uchovavat kontext testu tiez.